data/src/main/java/me/androidbox/data/local/DatabaseConstant.kt

```diff
@@ -0,0 +1,11 @@
+package me.androidbox.data.local
+
+object DatabaseConstant {
+    const val AGENDA_TABLE = "agenda_table"
+    const val FULL_AGENDA = "full_agenda"
+    const val ATTENDEE_TABLE = "attendee_table"
+    const val EVENT_TABLE = "event_table"
+    const val PHOTO_TABLE = "photo_table"
+    const val REMINDER_TABLE = "reminder_table"
+    const val TASK_TABLE = "task_table"
+}
```

Comment:
This still contains the full agenda and agenda table which isn't needed

Reply:
Here, Photo table and Attendee table are also optional, since you can save them as json